examples/mass_bias_from_models.ipynb

```diff
@@ -0,0 +1,527 @@
+    "- Repeat for NFW and Einasto\n",
+    "- plot the bias on the reconstructed mass as a function of the input bias"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def model_ref_nfw(r, logm, bias):\n",
+    "    m = 10.**logm\n",
+    "    tmp_Sigma = ccl_nfw_ana.projected (cosmo_ccl, r / a, m, a, mdef) / a**2\n",
+    "    tmp_BarSigma = ccl_nfw_ana.cumul2d (cosmo_ccl, r / a, m, a, mdef) / a**2\n",
+    "    model = (tmp_BarSigma - tmp_Sigma)*(1+bias)\n",
+    "    return model\n",
+    "\n",
+    "def model_ref_ein(r, logm, bias):\n",
+    "    m = 10.**logm\n",
+    "    nc_ein.props.log10MDelta = logm\n",
+    "    return np.array (smd.sigma_excess_array (nc_ein, cosmo, r, 1.0, 1.0, z))*(1+bias)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "logmass_nfw = []\n",
+    "logmass_ein= []\n",
+    "bias = np.logspace(-6,np.log10(0.5),30)\n",
+    "for b in bias:\n",
+    "    popt1,pcov1 = fitters['curve_fit'](lambda rr, logm:model_ref_nfw(rr, logm, b), \n",
+    "                        r, \n",
+    "                        ccl_DeltaSigma_nfw_ana, \n",
+    "                        np.zeros(len(r))+1.e-30, bounds=[13.,17.])\n",
+    "    popt2,pcov2 = fitters['curve_fit'](lambda rr, logm:model_ref_ein(rr, logm, b), \n",
+    "                        r, \n",
+    "                        nc_DeltaSigma_ein, \n",
+    "                        np.zeros(len(r))+1.e-30, bounds=[13.,17.])\n",
+    "\n",
+    "    logmass_nfw.append(popt1[0])\n",
+    "    logmass_ein.append(popt2[0])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "reldiff_nfw = np.abs(10**np.array(logmass_nfw) - Mvir)*100./Mvir\n",
+    "reldiff_ein = np.abs(10**np.array(logmass_ein) - Mvir)*100./Mvir\n",
+    "plt.scatter((bias)*100, reldiff_nfw, color='orange')\n",
+    "plt.scatter((bias)*100, reldiff_ein, color='blue', marker='+')\n",
+    "plt.plot(bias*100, 2.4*bias*100) # \"Fit by eye\"\n",
+    "plt.xscale('log')\n",
+    "plt.yscale('log')\n",
+    "plt.xlabel(r'Hypothetical bias on $\\Delta\\Sigma$ [%]')\n",
+    "plt.ylabel(r'Bias on reconstructed mass [%]')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## More realistic test for NFW\n",
+    "- Use the same set of ideal NFW data points, generated from the analytical CCL prescription\n",
+    "- Adjust the mass using the analytical and numerical NFW implementation of all backends\n",
+    "- Print the mass bias"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define all the models"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# NC numerical \n",
```

Comment:
# CCL numerical?
NC numerical is defined down below

Reply:
Yes, thanks and corrected.